controller/src/run.rs

```diff
@@ -14,6 +16,35 @@ use std::future::Future;
 use std::pin::Pin;
 use std::thread;
 
+/// Receive UpdateBackendStateMessages over core NATS, and turn them into
+/// BackendStateMessages over JetStream.
+pub async fn update_backend_state_loop(nc: TypedNats) -> NeverResult {
+    let mut sub = nc
+        .subscribe(UpdateBackendStateMessage::subscribe_subject())
+        .await?;
+    loop {
+        let msg = sub
+            .next()
+            .await
+            .ok_or_else(|| anyhow!("UpdateBackendStateMessage subscription ended."))?;
+        let value = msg.value.clone();
+
+        let value = BackendStateMessage {
+            backend: value.backend,
+            state: value.state,
+            cluster: Some(value.cluster),
+            time: value.time,
+        };
+
+        if let Err(e) = nc.publish_jetstream(&value).await {
+            tracing::error!(error=%e, "Failed to publish backend state message.");
```

Comment:
Do we want to retry if publishing fails?

Reply:
I've added a `continue` so that if there is a jetstream error, we do not ack, and the drone's retry loop kicks in.